contrib/net/http/client/client.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package client provides [context.Context]-aware alternatives to the
+// short-hand request functions [http.Get], [http.Head], [http.Post], and
+// [http.PostForm]. Using these functions allows for better control over the
+// trace context propagation.
+package client
```

Comment:
could this package be made internal? it doesn't seem to make a lot of sense outside of Orchestrion right?

Reply:
what I mean is this package doesn't provide any tracing specific funcionality, these are just convenience helpers over `http.NewRequestWithContext` + `http.DefaultClient.Do` that IMO shouldn't be part of the exposed api of dd-trace-go.